ursula_cli/shell.py

```diff
@@ -527,6 +542,14 @@ def parse_args():
         '--heat-stack-name', default=None,
         help='Name of the heat stack when heat provisioner is used',
     )
+    parser.add_argument(
+        '--heat-stack-update', default=False, action='store_true',
+        help='Update the heat stack',
+    )
+    parser.add_argument(
+        '--heat-parameters', default=None,
+        help='Semicolon separated list of KEY=VALUE parameters for heat',
```

Comment:
Yeah, that makes sense. I'll change it to a list and alter the help accordingly.